src/tacticsuggestions.ts

```diff
@@ -115,6 +115,16 @@ export class TacticSuggestions implements Disposable, CodeActionProvider {
         const lastPos = new Position(endLine, endCol)
         textEditor.selection = new Selection(lastPos, lastPos)
 
+        // Now check for `by ` before the tactic call, 
```

Comment:
```suggestion
        // Now check for `by ` before the tactic call,
```
This should fix the linter complaint.